app/src/modules/sparql.js

```diff
@@ -1,17 +1,40 @@
 const SPARQL_ENDPOINT = 'https://materialsmine.org/wi/sparql'
 
 async function querySparql (query, endpoint = SPARQL_ENDPOINT) {
-  const urlEncodedQuery = `${endpoint}?query=${encodeURIComponent(query)}&output=json`
+  const urlEncodedQuery = `${endpoint}?query=${encodeURIComponent(
+    query
+  )}&output=json`
   const requestOptions = {
     headers: {
       accept: 'application/sparql-results+json'
     }
   }
   return await fetch(urlEncodedQuery, requestOptions)
-    .then(async res => {
+    .then(async (res) => {
       const results = await res.json()
       return results
-    }).catch(err => console.log(err))
+    })
+    .catch((err) => console.log(err))
 }
 
 export { querySparql }
+
+export function parseSPARQL (response) {
+  const queryResults = []
```

Comment:
@aswallace would you help me test these changes against your existing code to make sure everything still works as it should? The majority are linter changes except for the new parseSparql

Reply:
Sounds good, thank you!